.github/workflows/ci.yml

```diff
@@ -79,11 +80,35 @@ jobs:
     if: github.event_name == 'pull_request'
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # checkout all commits to be able to determine merge base
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 17
+          cache: 'maven'
+      - name: Fetch base ref to find merge-base for GIB
+        run: .github/bin/git-fetch-base-ref.sh
       - name: Check Commits
         uses: trinodb/github-actions/block-commits@c2991972560c5219d9ae5fb68c0c9d687ffcdd10
         with:
           action-merge: fail
           action-fixup: none
+      - name: Check if all intermediate commits of this PR compile
+        run: |
+          git config user.name "Compile all commits builder"
+          git config user.email "compile-all-commits@trino.org"
```

Comment:
What's the purpose of this?

BTW, compile-all-commits@trino.org is not a valid email.

Reply:
`git rebase` needs an identity to tag rebased commits with.
These commits will be discarded at the end, but the important part is that `git rebase` will run whatever is passed in `--exec` between all commits

Reply:
I'm not sure if git validates this, but I finally noticed you probably meant that `trino.org` is not the Trino domain `trino.io`

Reply:
git doesn't validate it - any non-empty value works.